templates/vue/src/components/node-review/ReviewLog.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <ul>
+    <li v-for="event in eventWithComponents" :key="event.timestamp">
```

Comment:
I think the same key might be used for both the StatusChangeEvent and CommentEvent.
![image](https://user-images.githubusercontent.com/45089842/101834158-d380e780-3aee-11eb-913b-4bc8db5f3520.png)
![image](https://user-images.githubusercontent.com/45089842/101834684-881b0900-3aef-11eb-8882-ad109b717331.png)